opm/simulators/flow/BlackoilModelNldd.hpp

```diff
@@ -222,6 +241,9 @@ class BlackoilModelNldd
         for (const int domain_index : domain_order) {
             const auto& domain = domains_[domain_index];
             SimulatorReportSingle local_report;
+            if (std::find(domains_to_skip_.begin(), domains_to_skip_.end(), domain.index) != domains_to_skip_.end()) {
```

Comment:
It is not a priori clear to me that it is a good idea to skip these domains of isolated cells, as a local solve might reduce newton iterations. However, given that this code has been tested thoroughly, I assume the skipping approach has been validated?

Could store a flag in the domain itself rather than doing a lookup here, to save a little time.